db/system_distributed_keyspace.cc

```diff
@@ -543,7 +543,7 @@ system_distributed_keyspace::create_cdc_desc(
         co_await _sp.mutate(
             { std::move(m) },
             quorum_if_many(ctx.num_token_owners),
-            db::timeout_clock::now() + 10s,
+            db::timeout_clock::now() + 30s,
```

Comment:
I hope we don't get into a situation where we livelock without the ability to complete the operation. In theory we could increase the delta after a failure and retry, but this would be never-tested code. Let's hope this is sufficient.

In the raft era, topology changes will be atomic with cdc changes (cdc stream routing is just an aspect of topology).